tests/test_cluster.go

```diff
@@ -282,7 +283,9 @@ func NewClusterWithPersistenceTestBaseFactory(t *testing.T, options *TestCluster
 		}
 	}
 
-	taskCategoryRegistry := temporal.TaskCategoryRegistryProvider(archiverBase.metadata)
+	dcClient := dynamicconfig.StaticClient{dynamicconfig.CallbackProcessorEnabled: true}
+	dcc := dynamicconfig.NewCollection(dcClient, log.NewNoopLogger())
+	taskCategoryRegistry := temporal.TaskCategoryRegistryProvider(archiverBase.metadata, configs.NewConfig(dcc, 0))
```

Comment:
I think there's a central place for those overrides.
https://github.com/temporalio/temporal/blob/34862c6d10d45611dacef346b34e19a0c6b0be01/tests/dynamicconfig.go#L42

Reply:
Doesn't seem like there's a way to access this client during setup but I did change this to allow overriding via DynamicConfigOverrides for use in the test suites.